code/game/objects/items/weapons/implant/implants/carrion/explosive_spider.dm

```diff
@@ -2,9 +2,32 @@
 	name = "explosive spider"
 	icon_state = "spiderling_explosive"
 	spider_price = 40
+	var/devastation_range = -1
+	var/heavy_range = 0
+	var/weak_range = 2
+	var/flash_range = 6
+	var/det_time = 20
```

Comment:
You could change this to time defines like `2 SECONDS`.

Reply:
made it that way so it resembles grenades timer, will change